Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs

```diff
@@ -916,7 +916,7 @@ public void UpdateAppearance(Entity<StorageComponent?, AppearanceComponent?> ent
         }
 
         if (TryComp<StorageComponent>(insertEnt, out var insertStorage)
-            && GetMaxItemSize((insertEnt, insertStorage)) >= maxSize
+            && GetMaxItemSize((insertEnt, insertStorage)) > maxSize
```

Comment:
This check is for nested storages so you don't get infinite storage, 13 has a similar check.
For this we just have IgnoreContentsSize to override it when we know that it's safe, that's the preferred solution

Reply:
I see, I'll change other stuff to make sure to change the other stuff to use that then